runner/browserrunner.ts

```diff
@@ -0,0 +1,211 @@
+/**
```

Comment:
In hindsight that would have been a good idea. I just spent a while trying to rebase on top of an initial change like that and.. git got pretty unhappy.